### PR TITLE
bump github.com/elastic/elastic-integration-corpus-generator-tool to …

### PR DESCRIPTION
**Pull request by aspacca**
…v0.4.1

There was a bug in the corpus generator v0.4.0 that prevented corpora for datasets using the `awsAZFromRegion` function to be generated successfully

The bug was fixed in v0.4.1 and this PR bumps to that version

This PR changes as well the `benchmark generate-corpus` subcomand
Command example:
```
./elastic-package benchmark generate-corpus --commit 5ca23226adfe1b76869c5a1abc9428327b1a3a0b --dataset sqs --package aws --size 10K
```

The `--data-stream` flag was replaced with the more proper `--dataset`

It is now possible to generate directly a rally track:
```
./elastic-package benchmark generate-corpus --commit 5ca23226adfe1b76869c5a1abc9428327b1a3a0b --dataset sqs --package aws --size 10K --rally-track-output-dir ./track-folder
```

cc @ruflin

### CODE REVIEW

File: cmd/benchmark.go [hunk 1]
```diff
@@ -239,9 +239,9 @@ func generateDataStreamCorpusCommandAction(cmd *cobra.Command, _ []string) error
 		return cobraext.FlagParsingError(err, cobraext.PackageFlagName)
 	}
 
-	dataStreamName, err := cmd.Flags().GetString(cobraext.GenerateCorpusDataStreamFlagName)
+	dataStreamName, err := cmd.Flags().GetString(cobraext.GenerateCorpusDataSetFlagName)
 	if err != nil {
-		return cobraext.FlagParsingError(err, cobraext.GenerateCorpusDataStreamFlagName)
+		return cobraext.FlagParsingError(err, cobraext.GenerateCorpusDataSetFlagName)
```
Comment (mrodm) [inline cmd/benchmark.go:244]: nit, should it be renamed to `dataSetName` ?

**Review by mrodm — COMMENTED** [review]
LGTM
Just a comment about renaming the variable used for the dataset parameter.

**Comment by aspacca** [comment]
@mrodm please, look at the latest commits
thanks

File: cmd/benchmark.go [hunk 2]
```diff
@@ -28,10 +28,10 @@ import (
 )
 
 const generateLongDescription = `
-BEWARE: this command is in beta and it's behaviour may change in the future.
+*BEWARE*: this command is in beta and it's behaviour may change in the future.
 Use this command to generate benchmarks corpus data for a package.
 Currently, only data for what we have related assets on https://github.com/elastic/elastic-integration-corpus-generator-tool are supported.
-`
+For details on how to run this command, review the [HOWTO guide](./docs/howto/generate_corpus.md).`
```
Comment (mrodm) [inline cmd/benchmark.go:34]: :+1:

**Review by mrodm — APPROVED** [review]
LGTM!
Thanks for updating the readme generator to include the subcommands! :+1:

**Pull request merged**